packages/nuxt/package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "@vuestic-ui/nuxt",
```

```suggestion
  "name": "@vuestic/nuxt",
```

Same as `@vuestic/ag-grid-theme`